ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installation-nightly-4.18.yaml

```diff
@@ -56,6 +56,51 @@ resources:
       cpu: 400m
       memory: 800Mi
 tests:
+- as: aws-ipi-byo-iam-profile-master-f28
```

Comment:
I think maybe we can do some similar things like `azure-ipi-confidential-trustedlaunch` in regular prow ci jobs. 

Enable byo IAM profile on cp and compute in e2e job, and Enable byo IAM profile on default machine platform in destructive e2e job. 

If you want to cover byo-iam-on-only-cp (case-1), byo-iam-on-only-compute (case-2), byo-iam-on-default-machine (case-3), then we can re-use the existing regular ci jobs to cover case-1 and case-2, here we only need to add one more job to cover case-3. WDYT?

Reply:
e2e `cp+compute`, destructive e2e `default`
installation jobs: `cp-only` and `compute-only`
```
- as: aws-ipi-byo-iam-profile-arm-f7
  cron: 6 11 6,15,22,29 * *
  steps:
    cluster_profile: aws-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
      COMPUTE_NODE_TYPE: m6g.xlarge
      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
      OCP_ARCH: arm64
    test:
    - chain: openshift-e2e-test-qe
    workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-profile
- as: aws-ipi-byo-iam-profile-fips-amd-f28-destructive
  cron: 55 14 22 * *
  steps:
    cluster_profile: aws-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
      ENABLE_BYO_IAM_PROFILE_CONTROL_PLANE: "false"
      ENABLE_BYO_IAM_PROFILE_CUMPUTE: "false"
      ENABLE_BYO_IAM_PROFILE_DEFAULT_MACHINE: "true"
      FIPS_ENABLED: "true"
    test:
    - chain: openshift-e2e-test-qe-destructive
    workflow: cucushift-installer-rehearse-aws-ipi-byo-iam-profile
```

Reply:
looks good to me.

@yunjiang29 shall we cover them in 4.17 ?

Reply:
417 https://github.com/openshift/release/pull/58383